test/fixtures/dialects/ansi/create_table_constraint_reference_option.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE b (
+    b INT NOT NULL,
+    c INT NOT NULL,
+    d INT NOT NULL,
+    CONSTRAINT c_b FOREIGN KEY (b) REFERENCES a(b) ON DELETE RESTRICT ON UPDATE NO ACTION,
+    CONSTRAINT c_d FOREIGN KEY (d) REFERENCES a(d) ON UPDATE RESTRICT ON DELETE NO ACTION,
+    CONSTRAINT c_c FOREIGN KEY (c) REFERENCES a(c) ON DELETE RESTRICT
+)
```

Comment:
```suggestion
    CONSTRAINT c_b FOREIGN KEY (b) REFERENCES a(b) ON DELETE RESTRICT ON UPDATE NO ACTION,
    CONSTRAINT c_d FOREIGN KEY (d) REFERENCES a(d) ON UPDATE CASCADE ON DELETE SET NULL,
    CONSTRAINT c_c FOREIGN KEY (c) REFERENCES a(c) ON DELETE SET DEFAULT
)
```
Can we mix up the options here to test all of them?

Reply:
Yes we can.